src/BlockEntities/FurnaceEntity.cpp

```diff
@@ -173,6 +175,23 @@ bool cFurnaceEntity::ContinueCooking(void)
 
 
 
+int cFurnaceEntity::GetReward(void)
```

Comment:
The name of this function would be better as `GetAndResetReward`, it's not obvious that the reward counter is reset after it is called.